python/data_structures/queue.py

```diff
@@ -1,7 +1,7 @@
 import sys
 
-from data_structures.linked_list import Node, LinkedList
-from data_structures.invalid_operation_error import InvalidOperationError
+from linked_list import Node, LinkedList
+from invalid_operation_error import InvalidOperationError
```

Comment:
This change is breaking your tests.

I know that intuitively, this seems like it should work since these files are all in the same folder, but because we're actually referring to Python modules in our codebase here, we need to follow some rules. There are 2 options for formatting these imports:

```Python
# absolute import paths from the root, navigates into the correct module
from data_structures.linked_list import Node, LinkedList
from data_structures.invalid_operation_error import InvalidOperationError

# relative import paths up from where we are (I think). Since we are staying in the same directory, we must start with a dot (.)
from .linked_list import Node, LinkedList
from .invalid_operation_error import InvalidOperationError
```

Personally, I prefer to always use absolute imports, as I often get confused by relative pathing, but use whatever makes the most sense to you.

Reply:
Thanks Joseph!